docs/internal.rst

```diff
@@ -420,3 +420,110 @@ An example can be found in DTS parser:
 
 Setting `requestAutomaticFQTag` to `TRUE` implies setting
 `useCork` to `TRUE`.
+
+Multi passes parsing over multi input files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main part of Universal-ctags provides APIs for Multi passes
+parsing over Multi source files (MM). The main part applies
```

Comment:
Don't understand the meaning of "MM". First "M" is for multi, but what is the second for?
Typically you would do something like: "Multi source Files (MF)". That is, capitalize the words used in the abbreviation.